src/main/scala/org/ergoplatform/nodeView/wallet/persistence/WalletRegistry.scala

```diff
@@ -273,22 +269,45 @@ class WalletRegistry(store: HybridLDBKVStore)(ws: WalletSettings) extends Scorex
   /**
     * Updates scans of a box stored in the wallet database,
     * puts the box into the database if it is not there
+    * removes the box from the database if its there and scanIds are empty
     *
-    * @param scanIds
+    * @param newScans
     * @param box
     * @return
```

Comment:
Please add additional details about params and return or cleanup the declarations.

Reply:
Done, added descriptions